scripts/deb-install.sh

```diff
@@ -32,8 +32,9 @@ for cmd in wget gunzip; do
   command -v $cmd >/dev/null || { bb_log_err "$cmd command not found" && exit 1; }
 done
 
-# setup repository
-deb_setup_mariadb_mirror "$master_branch"
+# setup repository //TEMP this should not be needed since installation is
+# supposed to be done directly from artifacts, but maybe it is for dependencies?
+# deb_setup_mariadb_mirror "$master_branch"
```

Comment:
can you check the DEVELOPMENT_BRANCH variable to see if it's used in different places to do the same?

I remember the following setup: even if we install from the debs from ci.mariadb.org, we need galera which we installed from the mirror. Now, for the unreleased branches, this shouldn't exist, so I assume this is a good use case for DEVELOPMENT_BRANCH. Or do we install galera from ci.mariadb.org now as well?